include/mbedtls/ssl.h

```diff
@@ -1936,6 +1960,11 @@ void mbedtls_ssl_conf_authmode( mbedtls_ssl_config *conf, int authmode );
 */
 void mbedtls_ssl_tls13_conf_early_data( mbedtls_ssl_config *conf,
                                         int early_data_enabled );
+
+#if defined(MBEDTLS_SSL_CLI_C)
+int mbedtls_ssl_set_early_data( mbedtls_ssl_context* ssl, const unsigned char* buffer,
```

Comment:
We are not going to do it that way but by adding a new `mbedtls_ssl_write_early_data()` API. Thus please remove this.

Reply:
So, can we use the API mbedtls_ssl_write_early_data() when solving the issue #6340?

Reply:
Yes we will add the new API `mbedtls_ssl_write_early()` when working on #6340. Before, no need to write/read early data, we just exchange early data extensions.